test/parallel/test-assert.js

```diff
@@ -378,7 +378,7 @@ try {
 assert.strictEqual(true, threw,
                    'a.doesNotThrow is not catching type matching errors');
 
-assert.throws(function() { assert.ifError(new Error('test error')); });
+assert.throws(function() { assert.ifError(new Error('test error')), /^([A-Za-z])\w+$/});
```

Comment:
In this repo we try to make the validation of the error message to be as concrete as possible. For example:

```js
assert.throws(
  () => {
    throw new Error('Wrong value');
  },
  /^Error: Wrong value$/ // Instead of something like /Wrong value/
);
```

There is an ongoing PR to update our guide on this: https://github.com/nodejs/node/pull/11150